.circleci/config.yml

```diff
@@ -247,8 +245,7 @@ workflows:
                 - gpu.nvidia.small.multi
               with-mpi: [0]
               image-type:
-                - deepspeed
-                - gpt-neox-deepspeed
+                - deepspeed-gpt-neox
```

Comment:
why do we have deepspeed and pytorch-ngc images separate from cpu and pytorch-cuda images above? it's not gpu usage...

Reply:
Deepspeed needs a CUDA gpu to compile

Reply:
And pytorch-ngc images contain deepspeed